packages/common/lib/types/OpenID4VCIVersions.types.ts

```diff
@@ -0,0 +1,10 @@
+export enum OpenId4VCIVersion {
+  VER_9 = 'version9',
```

Comment:
Please assign ints, so we can do things like `if (ver > VER_9.valueOf())`

Reply:
fixed